configs/components/ruby.rb

```diff
@@ -143,6 +156,7 @@
     # This tells the ruby setup that it can use the default system gcc rather
     # than our own.
     target_dir = File.join(settings[:libdir], "ruby", "2.1.0", rbconfig_info[settings[:platform_triple]][:target_double])
+    sed = "sed"
     sed = "gsed" if platform.is_solaris?
```

Comment:
This is sort of meh but I know you didn't make the mess. Can we clarify intent a little with something like:

```ruby
sed = platform.is_solaris? "gsed" : "sed"
```